e3sm_diags/plot/utils.py

```diff
@@ -332,6 +296,89 @@ def _make_lon_cyclic(var: xr.DataArray):
     return new_var
 
 
+def _get_c_levels_and_norm(
```

Comment:
@forsyth2 all of the reusable plot utility functions start here at `_get_c_levels_and_norm()`.